VideoLocker/res/layout/row_discussion_comment.xml

```diff
@@ -16,32 +17,37 @@
         android:textSize="@dimen/edx_x_small"
         tools:text="This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment." />
 
-    <TextView
-        android:id="@+id/discussion_comment_count_report_text_view"
-        android:layout_width="wrap_content"
+    <LinearLayout
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentEnd="true"
-        android:layout_below="@id/discussion_comment_body"
-        android:background="?android:selectableItemBackground"
-        android:clickable="true"
-        android:drawablePadding="5dp"
-        android:padding="@dimen/discussion_responses_box_padding"
-        android:textColor="@color/edx_grayscale_neutral_base"
-        android:textSize="@dimen/edx_xxx_small"
-        tools:text="@string/discussion_responses_report_label" />
+        android:gravity="center"
```

Comment:
I don't think we need vertical centered gravity, because `LinearLayout` aligns the baselines of `TextView` and other views by default.